rules/cohort_slivar.smk

```diff
@@ -192,6 +194,8 @@ rule slivar_tsv:
             --gene-description {input.clinvar_lookup} \
             --gene-description {input.phrank_lookup} \
             --ped {input.ped} \
-            --out {output.comphet_tsv} \
-            {input.comphet_vcf}) > {log} 2>&1
+            --out /dev/stdout \
```

Comment:
Perhaps just drop the `--out` param instead of having `--out /dev/stdout` since default is to write to stdout.